src/passes/Inlining.cpp

```diff
@@ -607,24 +609,11 @@ struct FunctionSplitter {
   // Check if we can split a function. Returns whether we can. If the out param
   // is provided, also actually does the split, and returns the inlineable split
   // function in that out param.
-  bool maybeSplit(Function* func, Function** inlineableOut = nullptr) {
+  bool maybeSplit(Function* func) {
```

Comment:
It looks like this function no longer does splitting, so "maybe split" might be confusing. How about `canSplit`? Looks like it returns true iff we can. Or perhaps `computeSplit` as it computes the split kind and stashes that.

Reply:
Looks like this was addressed in a later commit.